shared/pingpong/pingpong.go

```diff
@@ -540,7 +547,16 @@ func constructTxn(from, to string, fee, amt, aidx uint64, client libgoal.Client,
 	}
 
 	if cfg.NumApp > 0 { // Construct app transaction
-		txn, err = client.MakeUnsignedAppNoOpTx(aidx, nil, nil, nil, nil)
+		// generate random assetID or appId if we send asset/app txns
+		accounts := cinfo.OptIns[aidx]
+		if len(accounts) > 0 {
+			limit := len(accounts) - 4
+			if limit > 0 {
+				start := rand.Intn(limit)
```

Comment:
for this test it would probably be good enough, but note that 4 consecutive accounts won't give us random access testing. maybe use rand.Perm and pick the first four ?

Reply:
done